### PR TITLE
Web application user interface

### PR DESCRIPTION
**Pull request by ahmetkudu**
Web application UI is created. This is a dynamic UI so everyone can locate calling API-related things. 
Also, as my task, get and post methods are used to request a world time API service.

### CODE REVIEW

**Comment by erkamkavak** [comment]
Nice work. Can you delete .idea folder and .DS_Store file because it seems like they are not related to the project but they are related more about your IDE settings. Also can you put a .gitignore file into project to prevent these files to be added to git. After that I will merge this pull request.

**Comment by ahmetkudu** [comment]
> Nice work. Can you delete .idea folder and .DS_Store file because it seems like they are not related to the project but they are related more about your IDE settings. Also can you put a .gitignore file into project to prevent these files to be added to git. After that I will merge this pull request.

I'm not sure to delete them. Because it wasn't the files I added directly. Either GitHub added itself or they were created as hidden in the project. If you are sure, I will delete it. Otherwise, they don't look like things that will cause problems.

File: practice-app/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+altgraph @ file:///System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot7/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/python3/python3-133.100.1.1/altgraph-0.17.2-py2.py3-none-any.whl
```
Comment (omertalib42) [inline practice-app/requirements.txt:1]: This dependency logic does not work as it searches the local drive, rather then the PyPi. Thus, project setup will not work in others.

It should be in the form of
altgraph==x.y.z (0.17.2, for this case)

File: practice-app/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+blinker==1.6.2
+certifi==2022.12.7
+charset-normalizer==3.1.0
+click==8.1.3
+Flask==2.3.1
+future @ file:///System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot7/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/python3/python3-133.100.1.1/future-0.18.2-py3-none-any.whl
```
Comment (omertalib42) [inline practice-app/requirements.txt:7]: Same as above.

File: practice-app/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+idna==3.4
+importlib-metadata==6.6.0
+itsdangerous==2.1.2
+Jinja2==3.1.2
+macholib @ file:///System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot7/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/python3/python3-133.100.1.1/macholib-1.15.2-py2.py3-none-any.whl
```
Comment (omertalib42) [inline practice-app/requirements.txt:12]: Same as above.

File: practice-app/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+MarkupSafe==2.1.2
+requests==2.29.0
+six @ file:///System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot7/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/python3/python3-133.100.1.1/six-1.15.0-py2.py3-none-any.whl
```
Comment (omertalib42) [inline practice-app/requirements.txt:15]: Same as above.

**Review by omertalib42 — CHANGES_REQUESTED** [review]

File: practice-app/requirements.txt (hunk 1 above)
Comment (omertalib42) [reply practice-app/requirements.txt:1]: To elaborate a bit more:

When we use `some_package==0.3.5`, pip automatically searches for the package in the pypi.org (if you want to look, pypi.org/simple), then finds the package, then downloads the package.

However, it's not necessary that we're going to use a package from PyPi. In fact, we may have our own package somewhere in our computer (or a modification of a package in somewhere), and we may want to use it. 

To do so, one can simple add `@` and the insert the path that pip installer should search for that package.

In our example, pip looks for the `altgraph` package in the filesystem of the computer, specifically looks at the `system/volumes/.... path`. If it cannot find it, it throws an error. For this case, it's probably in someone's computer, but not mine (and probably most of us do not have that package in that location).

File: practice-app/requirements.txt (hunk 4 above)
Comment (ahmetkudu) [reply practice-app/requirements.txt:15]: That's nice contribution to the practice-app.

Comment (omertalib42) [reply practice-app/requirements.txt:15]: So does yours :)

Comment (erkamkavak) [reply practice-app/requirements.txt:15]: @ahmetkudu Can you remove these lines(1, 7, 12, 15) from requirements.txt?

**Comment by erkamkavak** [comment]
Everything seems fine. Nice work @ahmetkudu @omertalib42.

**Pull request merged**